full-source-install.yml

```diff
@@ -0,0 +1,35 @@
+# ansible-playbook -i hostname-or-ip, -u root --ask-become-pass full-source-install.yml
+---
+- hosts: all
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+    pulp_source_dir: "/pulp_source/pulp"
+    pulp_git_repo: https://github.com/pulp/pulp.git
```

Comment:
This var and git_repo for each plugin need to be documented somewhere. Maybe just the pretask, which I imagine would move when the playbook is moved.